### PR TITLE
fix(types): add basic support for `bind` in Sequelize.query() method

### PR DESCRIPTION
**Pull request by devoto13**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [x] Does `npm run test` or `npm run test-DIALECT` pass with this change (including linting)?
- [x] Does the description below contain a link to an existing issue (Closes #[issue]) or a description of the issue you are solving?
- [x] Have you added new tests to prevent regressions?
- [x] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
- [x] Did you follow the commit message conventions explained in [CONTRIBUTING.md](https://github.com/sequelize/sequelize/blob/master/CONTRIBUTING.md)?

### Description of change

The `bind` property described in the [docs](http://docs.sequelizejs.com/manual/raw-queries.html#bind-parameter) was missing in the type definitions.

### CODE REVIEW

File: types/lib/query-interface.d.ts [hunk 1]
```diff
@@ -44,6 +44,14 @@ export interface QueryOptions extends Logging, Transactionable {
     [key: string]: string | number | string[] | number[];
   } | string[];
 
+  /**
+   * Either an object of named parameter replacements in the format `:param` or an array of unnamed
+   * replacements to replace `?` in your SQL.
+   */
+  bind?: {
```
Comment (SimonSchick) [inline types/lib/query-interface.d.ts]: Can you create a type here to avoid duplication?

Comment (devoto13) [reply types/lib/query-interface.d.ts]: Sure, done.

**Comment by devoto13** [comment]
While upgrading our codebase from 4 to 5 I've noticed that some queries also bind `null` and arbitrary JSON (for PostgreSQL JSON field). Probably `bind` should have more relaxed type than  `replacements`. Something like `{ [key: string]: any } | any[]`. What do you think?

**Comment by SimonSchick** [comment]
Not `any` no, we can define a simple type which represents anything json serializable.

But for now `unknown` would be ok

**Pull request merged**